Clinical-Genomics/scout
# Fix panel udate bug

bug fix #1661

**How to test**:
1. Install it on scout stage and make sure you can update a gene panel (not the first one on the list) using the following options:
- [x] Add genes using the gene panel which was initially failing for the user
- [x] Replace genes using the gene panel which was initially failing for the user
- [x] Test add/replace genes in panel using the Scout demo panel

![image](https://user-images.githubusercontent.com/28093618/73351098-991faa80-428e-11ea-8ce6-dd4670a76c0b.png)

**Expected outcome**:
The functionality should be working

**Review:**
